src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -447,7 +447,7 @@ public function iShouldNotBeAbleToConfirmOrderBecauseDoNotBelongsToShippingCateg
 
         $response = $this->ordersClient->getLastResponse();
 
-        Assert::same($response->getStatusCode(), 400);
+        Assert::same($response->getStatusCode(), 422);
```

Comment:
I wonder if `422` status code is a proper one here, wdyt @lchrusciel ?

Reply:
422 means that we are skipping validation and handling an exception. I would say, that it would be better to keep 400